tabbycat/options/preferences.py

```diff
@@ -614,7 +614,13 @@ class UseSpeakerRanks(ChoicePreference):
         ('high-points', 'Require ranking speeches, ranks congruent with speaker scores'),
     )
     default = 'none'
-
+@tournament_preferences_registry.register
+class PreparationTime(IntegerPreference):
+    help_text = _("How long after motion release does the round start(In Minutes)")
+    verbose_name = _("Preparation Time")
+    section = debate_rules
+    name = 'preparation_time'
+    default = 0
```

Comment:
```suggestion
    help_text = _("How long, in minutes, after motion release does the round start (-1 to deactivate)")
    verbose_name = _("Preparation Time")
    section = debate_rules
    name = 'preparation_time'
    default = -1
```

We use the convention of "-1" being an "off switch" for some other preferences; we should use the same convention. "0" could indicate an immediate start.